ui/core/src/model/calculations.ts

```diff
@@ -40,11 +41,11 @@ export const CALCULATIONS_CONFIG: Readonly<Record<CalculationType, CalculationCo
     description: 'Last value',
   },
   FirstNumber: {
-    label: 'First number',
+    label: 'First #',
```

Comment:
I need to check with design about how to shorten these. The word number is just too long to keep the columns small in the table.

Reply:
Design decision from @foxbat07: use an asterisk. Pushed up a change for that.